scripts/msb-tests.groovy

```diff
@@ -0,0 +1,63 @@
+node("cirhos_rhel7"){
+      deleteDir()
+
+      withEnv(["GOPATH=$WORKSPACE",
+               "BROKER_URL=http://msb-${NAMESPACE}.apps.${CLUSTER_ROUTE}"
+            ])
+            {
+                  stage('Clone the managed-service-broker repo'){
+                        dir(".") {
+                              git branch: 'master', url: 'https://github.com/integr8ly/managed-service-broker.git'
+                              if(!fileExists(file: './Makefile')) error("Error cloning managed-service-broker repo")
```

Comment:
I know I used to use this too but I think it is not required. In case of clone failure the `git` command fails with error, no need for explicit check afterwards.

Reply:
Yeah..copy/paste from you :-)